### ISSUE TITLE
Trouble accessing nested StringSlice

### ISSUE THREAD

**Issue by gregf** [spf13/viper#93]
I have the following yaml.

```yaml
filters:
  SomeTitle:
      reject:
         - "foo"
         - "bar"
      include:
         - "the"
         - "at"
```

I am trying to access the reject and include lists like so. viper.GetStringSlice("filters.SomeTitle.reject"), but I always end up with a empty slice.

**Comment by szuecs** [comment]
I have a similar issue.
If I request viper.Get("Bar") I get a map, but viper.Get("Bar.a") does not work.
In my usecase I would like to load the configuration which is a nested map, which does not work too. If I use a slice as base structure it works.

    viper.Marshal(&config)

json:

    Bar:
      a: foo
      b:
      - 1
      - 2

**Comment by szuecs** [comment]
I found my bug.

````
type Config struct {
	Entry map[string]ConfigEntry
}

type ConfigEntry struct {
	Name     string   // URL key to request, p.e. "foo" for http://macarque:8082/foo
	UrlFmt   string   // URL formatstring to request on every HostPort, p.e. "http://%s/foo.json"
	HostPort []string // Host Port string, p.e. ["web01:80", "web02:80"]
	Agg      string   // Name of the aggregation function, p.e. "sum"
	Key      string   // Key from json result to pull out the data to aggregate on
}

````

and the wrong yaml was:
````
  elected:
    hostPort:
     - web01:80
    - web02:80
    - web03:80
    agg: sum
    key: bar
  cpus_percent:
    hostPort:
    - web01:80
    - web02:80
    - web03:80
    agg: sum
    key: foo
````

the right yaml would be:
````
Entities:             <---- was missing
  elected:
    hostPort:
     - web01:80
    - web02:80
    - web03:80
    agg: sum
    key: bar
  cpus_percent:
    hostPort:
    - web01:80
    - web02:80
    - web03:80
    agg: sum
    key: foo
````

**Comment by jgsqware** [comment]
@gregf Have you fin d a way? I got the same problem

**Comment by gregf** [comment]
@jgsqware No I haven't.

**Comment by awishformore** [comment]
Fixed by PR #195
